app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1214,6 +1223,12 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         emptyMap()
     }
 
+    private val browserscreenCFRs: Map<BrowserScreenCfr, Boolean> get() = try {
+        FxNimbus.features.browserscreen.value().cfrsEnabled
+    } catch (e: NimbusFeatureException) {
```

Comment:
We don't need this try catch for NimbusFeatureException here because the docs are actually out of date. https://github.com/mozilla/application-services/issues/5085

Reply:
Thank you! Removed the try-catch.